runbot/runbot.py

```diff
@@ -1225,7 +1254,7 @@ def repo(self, repo=None, search='', limit='100', refresh='', **post):
         repo_ids = repo_obj.search(cr, uid, [])
         repos = repo_obj.browse(cr, uid, repo_ids)
         if not repo and repos:
-            repo = repos[0] 
+            repo = repos[0]
```

Comment:
I tried a lot of times, this still happen, Is this the git core.autocrlf problem? I set it to false or true is not working, or maybe the IDE problem? I use PYCHARM